src/java.desktop/share/classes/java/awt/MenuShortcut.java

```diff
@@ -46,7 +46,7 @@
  * using the {@code java.awt.event.KeyEvent.getExtendedKeyCodeForChar} call.
  * For example, a menu shortcut for "Ctrl+cyrillic ef" is created by
  * <p>
- * <code>MenuShortcut ms = new MenuShortcut(KeyEvent.getExtendedKeyCodeForChar('\u0444'), false);</code>
+ * <code>MenuShortcut ms = new MenuShortcut(KeyEvent.getExtendedKeyCodeForChar('ф'), false);</code>
```

Comment:
This is javadoc inJava SE specification. As is the Action case below.
I can't think of any actual harm from this change, so OK,  but I am not seeing why it is needed.